src/Models/Article.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace Astrogoat\Blog\Models;
+
+use Helix\Fabrick\Icon;
+use Helix\Lego\Models\Contracts\Indexable;
+use Helix\Lego\Models\Contracts\Metafieldable;
+use Helix\Lego\Models\Contracts\Searchable;
+use Helix\Lego\Models\Contracts\Sectionable;
+use Helix\Lego\Models\Model as LegoModel;
+use Helix\Lego\Models\Traits\HasMetafields;
+use Helix\Lego\Models\Traits\HasSections;
+use Spatie\Sluggable\HasSlug;
+use Spatie\Sluggable\SlugOptions;
+
+class Article extends LegoModel implements Sectionable, Indexable, Searchable, Metafieldable
```

Comment:
Where you able to add the `Publishable` contract and `CanBePublished` trait?

Reply:
Yes but just the issue we had the last time where the date change wasn't updating the Livewire state. I would want Abigail to proceed with QA of other features whilst I figure out how to fix that.